trio/__init__.py

```diff
@@ -13,86 +13,101 @@
 # This file pulls together the friendly public API, by re-exporting the more
 # innocuous bits of the _core API + the higher-level tools from trio/*.py.
 
+# Uses `from x import y as y` for compatibility with `pyright --verifytypes` (#2625)
+
 from ._version import __version__
```

Comment:
`as __version__`?

Reply:
huh, `pyright --verifytypes` seems to entirely ignore `__version__`. No matter how it's imported, or even if it's statically defined in the file. (but if I create a `__verison__` variable that one shows up with `--verbose`).

Reply:
see later conversation on trio.socket constants

Reply:
Aha, found the documentation for it:
https://github.com/microsoft/pyright/blob/main/docs/typed-libraries.md#type-completeness
> Type annotations can be omitted in a few specific cases where the type is obvious from the context:
> ...
> The following module-level symbols do not require type annotations: `__all__`,`__author__`, `__copyright__`, `__email__`, `__license__`, `__title__`, `__uri__`, `__version__`.